### DISCUSSION TITLE
Wide and big tiles

### DISCUSSION THREAD

**Discussion by Darkbluetest** [valinet/ExplorerPatcher#3751] [Ideas]
Hi,

I am using the last version of EP with W11 23H2, and the taskbar and start menu from W10. It works wonderfully, AND the live tiles work perfectly (ms weather, global clock tile, photos, etc), in any size, wide, big or medium.

It is really frustrating that apps like EdgeTile Creators or Live Tiles Anywhere, are not able to pin the tiles, though. I could understand that they would not work if ANY live tiles that worked in W10 did not work in EP, but this is not the case: the existing live tiles work still perfectly... So from my ignorance, technically I think that these apps should work, or at least, it should be possible to take their "output tiles" and make them work, in some way.

Alternatively, I wonder if it is not possible to find a way to make/build wide and big tiles in/for EP W10 SM... It should be possible as they still work. It seems that it could be the "installation approach" of these custom made live tiles:

Although it is related to another app, it could shed some light: the Files app (files.community), an explorer replacement, uses a different approach when you select the context menu option "Pin to Start Menu" than the vanilla W11 explorer/context menu. The Files one is not able to pin anything to EP W10 SM, but indeed the W11 file explorer, can pin the very same files to the W10 EP SM as tiles. The approach used to pin it seems to be the problem, and not the live tile itself.

I wonder if you see some way or possibility to solve this. Maybe there is some program out there that makes these tiles/installation as the other ones, which work ok, so compatible with EP? Could even be possible to make a little tool/or even a simple guide on how to make/install them (even in an hand made way) so they are compatible? I have not found really enough and clear information about how these live tiles work and where they should be stored/installed, etc.

If I am not wrong this is the only thing missing to have the W10 back at full. Tx!

**Comment by milestprower92** [comment]
The reason why apps can't pin tiles to the 10 start menu is because the pop up that asks if you want to pin the item to the start menu redirects the action to the 11 start menu

look at this discussion to [learn more about the issues present in the Windows 10 Start Menu](https://github.com/valinet/ExplorerPatcher/discussions/1679)

**Comment by Darkbluetest** [comment]
I wonder if it could not be possible to replace that dialog with the Windows 10 native one...as done with start menu (?)
